be/src/olap/tablet_schema.h

```diff
@@ -376,7 +382,15 @@ class TabletSchema : public MetadataAdder<TabletSchema> {
     void set_row_store_page_size(long page_size) { _row_store_page_size = page_size; }
     long row_store_page_size() const { return _row_store_page_size; }
 
-    const std::vector<TabletIndex>& indexes() const { return _indexes; }
+    const std::vector<const TabletIndex*> inverted_indexes() const {
+        std::vector<const TabletIndex*> inverted_indexes;
```

Comment:
It will cause allocate and free small vector memory.

Reply:
For the sake of code clarity, this is worthwhile.